hack/runtime-migrator/internal/runtime/migrator.go

```diff
@@ -145,10 +145,26 @@ func getAdministratorsList(ctx context.Context, provider kubeconfig.Provider, sh
 	})
 
 	subjects := make([]string, 0)
-
 	for _, clusterRoleBinding := range clusterRoleBindings.Items {
-		for _, subject := range clusterRoleBinding.Subjects {
-			subjects = append(subjects, subject.Name)
+		// We are interested only in cluster-admin role
+		if clusterRoleBinding.RoleRef.Kind == "ClusterRole" && clusterRoleBinding.RoleRef.Name == "cluster-admin" {
+			willMigrate := false
+			for _, subject := range clusterRoleBinding.Subjects {
+				// We are interested only in users
+				if subject.Kind == rbacv1.UserKind {
+					subjects = append(subjects, subject.Name)
+					willMigrate = true
+				}
+			}
+
+			if willMigrate {
```

Comment:
The logic should respect `dry-run` flag. The CRBs should be labeled only if `dry-run=false`